backport/backport.ts

```diff
@@ -238,13 +251,12 @@ const backport = async ({
 	sender,
 }: BackportArgs) => {
 	let labelsString = labels.map(({ name }) => name)
-	if (
-		!(
-			labelsString.includes('type/bug') ||
-			labelsString.includes('product-approved') ||
-			labelsString.includes('type/docs')
-		)
-	) {
+	let matchedLabels = getMatchedBackportLabels(labelsString, backportLabels)
+	let matches = false
+	for (const label of labelsString) {
+		matches = labelRegExp.test(label)
```

Comment:
doesn't this override `matches` for each label? so in the end `matches ` contains only whether the last label matched

Reply:
the same applies to the .js version

Reply:
ohoh right, we need to break if true.
good catch thanks @dimitarvdimitrov 🙇